packages/volto/razzle.config.js

```diff
@@ -417,6 +417,20 @@ const defaultModify = ({
         ]
       : [];
 
+  //prefix-path
+  const prefixPath = process.env.RAZZLE_PREFIX_PATH || '';
+  if (prefixPath) {
+    if (target === 'web' && dev) {
```

Comment:
TODO: prefix assets on prod only. rename vars

Reply:
@nileshgulia1 @sneridagh I still don't understand the advantage of the prefix working only in production mode. When we are developing the prefix functionality, it is necessary for it to work in development mode. For example, when we are fixing a Cypress test.